openeogeotrellis/utils.py

```diff
@@ -572,3 +574,85 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 @functools.lru_cache
 def is_package_available(name: str) -> bool:
     return any(m.name == name for m in pkgutil.iter_modules())
+
+
+def reproject_cellsize(
```

Comment:
this function deserves a couple of unit tests I think

Reply:
true, added under `test_reproject_cellsize(...)`